test/gtest/uct/ib/test_dc.cc

```diff
@@ -475,9 +475,47 @@ UCS_TEST_SKIP_COND_P(test_dc, stress_iface_ops,
 
 UCT_DC_INSTANTIATE_TEST_CASE(test_dc)
 
+static const char *uct_dc_tx_policy_names[] = {
+    [UCT_DC_TX_POLICY_DCS]           = "dcs",
+    [UCT_DC_TX_POLICY_DCS_QUOTA]     = "dcs_quota",
+    [UCT_DC_TX_POLICY_RAND]          = "rand",
```

Comment:
1. pls check clang format
2. IMO no need to test DCS, it's enough to test just DCS_QUOTA
3. maybe move it inside enum_resources function?